dollosa-michael/chess-flex/css/style.output.css

```diff
@@ -0,0 +1,87 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0; }
+
+html {
+  font-size: 62.5%;
+  /* 1rem = 10px; 10px/16px = 62.5% -- 16px is default font size in browser*/ }
+
+body {
+  background-image: url("../assets/bg.jpeg");
+  background-size: cover;
+  background-repeat: no-repeat; }
+
+.container {
+  background-color: #e6b7cd;
+  width: 80vmin;
+  height: 80vmin;
+  position: absolute;
+  top: 50%;
+  /* remember the the origin will be the relative position of the parent */
+  left: 50%;
+  /* remember the the origin will be the relative position of the parent */
+  transform: translate(-50%, -50%);
+  /* shifted in related to the current position of the element itself */
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  border: 1.5rem solid #e6b7cd;
+  border-radius: 2rem;
+  box-shadow: rgba(0, 0, 0, 0.07) 0px 1px 2px, rgba(0, 0, 0, 0.07) 0px 2px 4px, rgba(0, 0, 0, 0.07) 0px 4px 8px, rgba(0, 0, 0, 0.07) 0px 8px 16px, rgba(0, 0, 0, 0.07) 0px 16px 32px, rgba(0, 0, 0, 0.07) 0px 32px 64px; }
+
+.row {
+  background-color: transparent;
+  display: flex;
+  flex-wrap: wrap; }
+
+.box {
+  background-color: white;
+  width: 10vmin;
+  height: 10vmin;
+  box-shadow: 0 0 1rem 0 rgba(0, 0, 0, 0.2);
+  overflow: hidden;
+  position: relative;
+  font-size: 6rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center; }
+  @media only screen and (max-width: 700px) {
```

Comment:
good job on applying responsive in your work. keep it up.

Reply:
"what package did you install and use?"
- installed node-sass, npm-run-all, and concat lang sir